### ISSUE TITLE
Credo.Check.Readability.MaxLineLength not ignoring strings after certain sigils

### ISSUE THREAD

**Issue by SamuelWillis** [rrrene/credo#918]
### Precheck

* For bugs, please do a quick search and make sure the bug has not yet been reported here

### Environment

* Credo version (`mix credo -v`): 1.6.1
* Erlang/Elixir version (`elixir -v`):
```
  Erlang/OTP 24 [erts-12.0.2] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [jit]
  Elixir 1.12.2 (compiled with Erlang/OTP 24)
```
* Operating system: OSX BigSur 11.6

### What were you trying to do?

After upgrading to 1.6.1 I seeing some odd `MaxLineLength` failures for long strings on lines after `~L` and `~H` sigils.
The `MaxLineLength` will fail if the `~w` sigil is used before the string as well although I first saw this with a long string in a function after a LiveComponent's render function.

The long string does not need to be in a separate function either.

The following fails the `MaxLineLength` check:
```elixir
  def render(assigns) do
    ~H"""
    My render template
    """
  end

  def long_string do
    "This is a very long string that is after a ~H sigil, I would expect that it is ignored because I set the `ignore_strings` rule to be true."
  end
```

The following passes the `MaxLineLength` check:
```elixir 
  def long_string do
    "This is a very long string that is before a ~H sigil, it will pass the MaxLineLength check as expected because `ignore_strings` has been set to true."
  end

  def render(assigns) do
    ~H"""
    My render template
    """
  end
```

### Expected outcome
The long string would be ignored even if it is after a sigil as per `ignore_string: true`.

### Actual outcome
Long strings are failing checks if they appear after certain sigils.

**Comment by rrrene** [comment]
This seems odd. Thx for reporting! 👍

**Comment by rrrene** [comment]
Thx for reporting this! :+1: It should be be fixed in the latest version of Credo.

If it is not, please feel free to re-open this issue!
